config/manifests/bases/vault-secrets-operator.clusterserviceversion.yaml

```diff
@@ -72,6 +121,9 @@ spec:
     url: https://github.com/hashicorp/vault-secrets-operator
   - name: Documentation
     url: https://developer.hashicorp.com/vault/docs/platform/k8s/vso
+  maintainers:
+  - email: support@hashicorp.com
+    name: HashiCorp Support
```

Comment:
I'm not sure if this is the right email here, so I'm open to suggestions.

Reply:
Does it make sense that maintainers be whatever we have for CODEOWNERS for the repo?